Vocable/Features/Settings/EditCategories/EditCategoriesViewController.swift

```diff
@@ -6,9 +6,10 @@
 //  Copyright © 2020 WillowTree. All rights reserved.
 //
 
-import UIKit
 import Combine
 import CoreData
+import SwiftUI
+import UIKit
```

Comment:
Another unused import

```suggestion
import Combine
import CoreData
import UIKit
```